src/components/structures/TimelinePanel.js

```diff
@@ -1141,6 +1141,18 @@ class TimelinePanel extends React.Component {
     // get the list of events from the timeline window and the pending event list
     _getEvents() {
         const events = this._timelineWindow.getEvents();
+
+        // `slice` performs a shallow copy of the array
```

Comment:
`arrayFastClone` when you're in this layer please, so we can always keep the most performant option readily available